RoomDraw/app/models/occupy.rb

```diff
@@ -1,5 +1,4 @@
 class Occupy < ActiveRecord::Base
-
   # define primary key
   self.primary_key = [:student_id, :dorm_name, :room_num]
```

Comment:
Given we removed `dorm_name` and `room_num` as primary keys for the `Rooms` table, opting instead merely to use `room_id`, why isn't that what we're doing here?